extensions/wikia/UserLogin/UserLoginHooksHelper.class.php

```diff
@@ -116,7 +116,12 @@ public static function onSetUserEmail( User $user, $newEmail, &$result, &$info )
 	}
 
 	public static function isValidEmailAddr( $addr ) {
-		return preg_match( '/^[a-z0-9._%+-]+@(?:[a-z0-9\-]+\.)+[a-z]{2,4}$/i', $addr ) !== 0;
+		// this validation match Helios email validation
+		// [see: https://github.com/Wikia/helios/blob/ce490394ae10eb248afd2a063dc465d8eb4efb01/domain/user_validator_funs.go#L29]
+		return preg_match(
+			'/^[\w!#$%&\'*+\/=?^_`{|}~-]+(?:\.[\w!#$%&\'*+\/=?^_`{|}~-]+)*@(?:[\w](?:[\w-]*[\w])?\.)+[a-zA-Z0-9](?:[\w-]*[\w])?$/i',
+			$addr
+		) !== 0;
```

Comment:
@garthwebb ```filter_var``` is not compatible with regex used in Helios, see example: https://ideone.com/8tD9TZ so it won't fix initial problem. MW will still failing for those email addresses (See ticket description).

As you can see in SERVICES-796 it won't be fixed in near future, so for now I'd proceed with existing Helios regex to avoid problems in MW. When it gets fixed in Helios we you update our way of validating email addresses.